deidentify.gemspec

```diff
@@ -2,7 +2,7 @@
 
 Gem::Specification.new do |s|
   s.name        = 'deidentify'
-  s.version     = '2.5.0'
+  s.version     = '2.5.1'
```

Comment:
you also need to run `bundle install` to update the gemlock

Reply:
Gemfile.lock updated now Thanks @lucy-dement